src/client/package.json

```diff
@@ -32,6 +32,7 @@
     "test": "vitest",
     "verify": "npm run typecheck && npm run lint && npm run format:check && npm run test",
     "start": "cross-env TARGET=web vite",
+    "start:local": "cross-env VITE_LOCAL_HYDRA_URL=ws://localhost:8929 TARGET=web vite",
```

Comment:
these should be fine with VITE_HYDRA_URL=ws://localhost:8929 and no changes in `connection.ts` .. pretty sure I used this approach with recent local testing on prev laptop .. but yeah, not sure why I/we did not add a local script 😄